### DISCUSSION TITLE
Tab is not highlighted when it's active..

### DISCUSSION THREAD

**Discussion by antonioarmada** [huntabyte/shadcn-svelte#1249] [Help]
My tabs always remain the same gray background, regardless of whether they are focused or active. I've tried all the examples I found here, but there's still no change... 

I'm the only one with this issue?

<img width="272" alt="Captura de pantalla 2024-08-21 a la(s) 11 45 03" src="https://github.com/user-attachments/assets/523871b5-1ef7-4480-ad70-7e2b3bb8daf9">

**Comment by oriolmontcreus** [comment]
How does your current implementation look like?
Do you see any errors in your console?
Here's an example implementation:
```svelte
      <Tabs value="all" class="mb-4">
        <TabsList>
          <TabsTrigger value="all">All Projects</TabsTrigger>
          <TabsTrigger value="open-source">Open Source</TabsTrigger>
          <TabsTrigger value="in-progress">Closed Source</TabsTrigger>
        </TabsList>
      </Tabs>
```
